app/scripts/controllers/dashboard.ts

```diff
@@ -63,15 +76,27 @@ module labsFrontendApp {
             };
         }
 
+        private addTogglesToFeatureSections(toggles) {
+            this.$scope.feature_sections.push.apply(this.$scope.feature_sections, toggles)
+        }
+
         /**
          * @param releases
          */
-        getToggles( releases: ng.IPromise<Array<Release>> )
+        getUserToggles( releases: Array<Release> )
         {
-            releases.then( ( r: Array<Release> ) => {
-                this.toggles.execute( 1 ).then( ( toggles: Array<Toggle> ) => {
-                    this.$scope.feature_sections = toggles;
-                } );
+            this.userToggles.execute( releases[0].id ).then( ( toggles: Array<Toggle> ) => {
+                this.$scope.user_features = toggles;
```

Comment:
given that both services return Toggles they could be made to implement an interface to cut down on this code duplication